control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -1146,6 +1146,32 @@ paths:
           $ref: '#/components/responses/ServerError'
       security:
         - JWT: []
+  '/label/pools':
```

Comment:
hmm this doesn't fit here, we're not getting pools from labels.
I think this should be another parameter on the /pools itself

Reply:
done